docs/howto/django/tests.md

```diff
@@ -44,7 +43,7 @@ from procrastinate.contrib.django.models import ProcrastinateJob
 
 def test_my_task(app):
     # Run the task
-    app.defer("my_task", args=(1, 2))
+    task.defer(a=1, b=2)
```

Comment:
Maybe we should name it `my_task` and add a pseudo import at the top (e.g. `from mypackage.procrastinate import my_task` to make it clearer that this is the task defined by the user.